### PR TITLE
satellite/satellitedb: add default offer for offers table

### PR DESCRIPTION
**Pull request by VinozzZ**
What: 
marketing team needs two default offer for the referral program
Why:
add insert into migration so the two default offer is auto generated

Please describe the tests:
 - Test 1:
 - Test 2:
 
Please describe the performance impact:

## Code Review Checklist (to be filled out by reviewer)
 - [x] Does the PR describe what changes are being made?
 - [x] Does the PR describe why the changes are being made?
 - [x] Does the code follow [our style guide](https://github.com/storj/docs/blob/master/code/Style.md)?
 - [x] Does the code follow [our testing guide](https://github.com/storj/docs/blob/master/code/Testing.md)?
 - [x] Is the PR appropriately sized? (If it could be broken into smaller PRs it should be)
 - [x] Does the new code have enough tests? (*every* PR should have tests or justification otherwise. Bug-fix PRs especially)
 - [x] Does the new code have enough documentation that answers "how do I use it?" and "what does it do?"? (both source documentation and [higher level](https://github.com/storj/docs), diagrams?)
 - [x] Does any documentation need updating?
 - [x] Do the database access patterns make sense?

### CODE REVIEW

**Comment by VinozzZ** [comment]
> This isn't returning any offers for the `GET` handler

What do you mean?

**Comment by fuskovic** [comment]
@VinozzZ when I run this in my local environment and log out the result of `db.ListAll()` it doesn't return any offers. I was expecting to get two offers - one for each offer type.

**Comment by VinozzZ** [comment]
>  when I run this in my local environment and log out the result of `db.ListAll()` it doesn't return any offers. I was expecting to get two offers - one default offer for each offer type.
@fuskovic are you running storj-sim with postgres? if not, you won't see the change locally since storj-sim is using sqlite

File: satellite/rewards/rewards_test.go [hunk 1]
```diff
@@ -59,27 +59,26 @@ func TestOffer_Database(t *testing.T) {
...
-				ID:        new.ID,
-				Status:    rewards.Done,
-				ExpiresAt: time.Now(),
-			}
+			//TODO: delete this test after removing Redeem method
 
-			isDefault := update.Status == rewards.Default
-			err = planet.Satellites[0].DB.Rewards().Redeem(ctx, update.ID, isDefault)
-			require.NoError(t, err)
+			//isDefault := new.Status == rewards.Default
```
Comment (coyle) [inline satellite/rewards/rewards_test.go]: why not just delete now?

File: satellite/rewards/rewards_test.go [hunk 2]
```diff
@@ -59,27 +59,26 @@ func TestOffer_Database(t *testing.T) {
...
+			//current, err := planet.Satellites[0].DB.Rewards().ListAll(ctx)
+			//require.NoError(t, err)
+			//if current[i].Status == rewards.Default {
+			//	require.Equal(t, new.NumRedeemed, current[i].NumRedeemed)
+			//} else {
+			//	require.Equal(t, new.NumRedeemed+1, current[i].NumRedeemed)
+			//}
 
-			err = planet.Satellites[0].DB.Rewards().Finish(ctx, update.ID)
+			err = planet.Satellites[0].DB.Rewards().Finish(ctx, all[i].ID)
```
Comment (coyle) [inline satellite/rewards/rewards_test.go]: is it possible for `i` to be greater than the length of `all` ?

Comment (thepaul) [reply satellite/rewards/rewards_test.go]: Looks like no- we insert one offer in each iteration, so the length of "all offers" will be at least i+1

**Pull request merged**